guide/.directory

```diff
@@ -0,0 +1,4 @@
+[Dolphin]
```

Comment:
what is this file?

Reply:
No idea hahahaha. Probably Kubuntu things

Reply:
What about this point @Guillemdb 
* Mathematical formulas are not displayed correctly (section "python-intro")